doc/UsersGuide/source/FAQ.rst

```diff
@@ -0,0 +1,95 @@
+.. _FAQ:
+
+***
+FAQ
+***
+
+==============================================================
+How do I build and run a single test of the UFS Weather Model?
+==============================================================
+
+An efficient way to build and run the UFS Weather Model is to use the regression test
+(``rt.sh``).  This script is widely used by model developers on Tier 1 and 2 platforms
+and is described in :numref:`Section %s <ConductingRegTests>`.  The advantages to this approach are:
+
+- You can bypass the workflow, pre- and post-processing steps.
```

Comment:
I suggest changing the language here. The UFS WM does not have a workflow with pre- and post- that needs to be bypassed (the App does, but this documentation is for the model).
```suggestion
- It does not require a workflow, pre- or post-processing steps.
```